.github/workflows/test.yaml

```diff
@@ -24,3 +24,31 @@ jobs:
 
       - name: Lint
         run: yarn run lint
+
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          cache: yarn
+          node-version: '16'
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Test
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        run: |
+          npx playwright install --with-deps chromium
+          xvfb-run --auto-servernum yarn run test:ci
```

Comment:
Do we need to do `xvfb-run` anymore?

Reply:
Unfortunately for now. The headed mode seems to bring a higher degree of uncertainty to the tests in github actions. Stable on localhost though. Needs more investigation.